src/Markdig/Parsers/FencedBlockParserBase.cs

```diff
@@ -1,231 +1,241 @@
-            {
-                return BlockState.None;
-            }
-
-            // specs spaces: Is space and tabs? or only spaces? Use space and tab for this case
-            line.TrimStart();
-
-            var fenced = CreateFencedBlock(processor);
-            {
-                fenced.Column = processor.Column;
-                fenced.FencedChar = matchChar;
-                fenced.FencedCharCount = count;
-                fenced.Span.Start = startPosition;
-                fenced.Span.End = line.Start;
-            };
-
-            // Try to parse any attached attributes
-            if (TryParseAttributes != null)
-            {
-                TryParseAttributes(processor, ref line, fenced);
-            }
-
-            // If the info parser was not successfull, early exit
-            if (InfoParser != null && !InfoParser(processor, ref line, fenced))
-            {
-                return BlockState.None;
-            }
-
-            // Add the language as an attribute by default
-            if (!string.IsNullOrEmpty(fenced.Info))
-            {
-                if (string.IsNullOrEmpty(InfoPrefix))
-                {
-                    fenced.GetAttributes().AddClass(fenced.Info);
-                }
-                else
-                {
-                    fenced.GetAttributes().AddClass(InfoPrefix + fenced.Info);
-                }
-            }
-
-            // Store the number of matched string into the context
-            processor.NewBlocks.Push(fenced);
-
-            // Discard the current line as it is already parsed
-            return BlockState.ContinueDiscard;
-        }
-
-        protected abstract T CreateFencedBlock(BlockProcessor processor);
-
-        public override BlockState TryContinue(BlockProcessor processor, Block block)
-        {
-            var fence = (IFencedBlock)block;
-            var count = fence.FencedCharCount;
-            var matchChar = fence.FencedChar;
-            var c = processor.CurrentChar;
-
-            // Match if we have a closing fence
-            var line = processor.Line;
-            while (c == matchChar)
-            {
-                c = line.NextChar();
-                count--;
-            }
-
-            // If we have a closing fence, close it and discard the current line
-            // The line must contain only fence opening character followed only by whitespaces.
-            if (count <=0 && !processor.IsCodeIndent && (c == '\0' || c.IsWhitespace()) && line.TrimEnd())
-            {
-                block.UpdateSpanEnd(line.Start - 1);
-
-                // Don't keep the last line
-                return BlockState.BreakDiscard;
-            }
-
-            // Reset the indentation to the column before the indent
-            processor.GoToColumn(processor.ColumnBeforeIndent);
-
-            return BlockState.Continue;
-        }
-    }
+// Copyright (c) Alexandre Mutel. All rights reserved.

```

Comment:
Weird, why this file is being marked as completely modified?

Reply:
MB, apparently I let VS set \r\n line endings for this file.
Actual changes: [gist diff](https://gist.github.com/MihaZupan/2457ea2497f4e58fdd5616a8a75900bf/revisions#diff-44589aff4202ecb8017748d2d2e7a181)